.env

```diff
@@ -0,0 +1,7 @@
+export GOOGLE_APPLICATION_CREDENTIALS=".auth.json"
```

Comment:
this file probably doesn't need to be here either

Reply:
Deleted!